tests/stubs/openai.ts

```
@@ -0,0 +1,27 @@
+namespace OpenAI {
+        export interface ChatCompletionMessageParam {
+                role: string;
+                content: string;
+        }
+        export namespace Chat {
+                export namespace Completions {
+                        export interface ChatCompletionTool {
+                                type?: string;
+                                function?: { name: string; description?: string; parameters?: unknown };
+                        }
+                        export interface ChatCompletionToolChoiceOption {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)